README.md

```diff
@@ -1,2 +1,33 @@
-# safe-auth
-Versatile authentication system for Node
+[![Build Status](https://travis-ci.org/evenset/safe-auth-sequelize.svg?branch=development)](https://travis-ci.org/evenset/safe-auth-sequelize)
+[![codecov](https://codecov.io/gh/evenset/safe-auth-sequelize/branch/development/graph/badge.svg)](https://codecov.io/gh/evenset/safe-auth-sequelize)
+
+# What's safe-auth-sequelize
+It's [Sequelize](https://github.com/sequelize/sequelize) adapter for
+[safe-auth](https://github.com/evenset/safe-auth). So it enables Postgres,
+MySQL, MariaDB, SQLite and Microsoft SQL Server database engines for safe-auth
+as its storage backend.
+It's written in TypeScript with %100 test code coverage.
+
+# How to use
+
+It provides a function that should be called with an instance of sequelize:
+
+```typescript
+import safeAuthSequelize from 'safe-auth-sequelize';
+```
+
+It then returns a JS object that provides two models:
+
+```typescript
+const {SequelizeAccessToken, SequelizeUser} = safeAuthSequelize(sequelize);
+```
+
+Each of these two models implement the corresponding safe-auth model and
+Sequelize model at the same time, so they store stuff in the database the
+`sequelize` instance is connected to.
```

Comment:
do they have to run migration? how do they pass the user model name to it?

Reply:
@hesamd 
Sequelize doesn't prove us utilities to have a clean flow for migrations.
These are the options I can think of (from easy to hard):
1. We can provide the migration in the repository so that people manually download it in their migrations directory.
2. We can provide it in the npm package so that people import it in one of their migrations (extremely dangerous in case we change the migration in a newer version, it'll silently replace the old migration.)
3. We can provide utility functions like `safeAuthSequelizeMigrations.createTable('users')` and `safeAuthSequelizeMigrations.createColumn('users', 'username')`

Currently we're going with first approach. I'm adding it to the README.md

Reply:
@hesamd done